soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_item.c

```diff
@@ -333,12 +337,20 @@ void KaleidoScope_HandleItemCycles(PlayState* play) {
         play,
         SLOT_TRADE_CHILD,
         CanMaskSelect(),
-        INV_CONTENT(ITEM_TRADE_CHILD) <= ITEM_MASK_KEATON || INV_CONTENT(ITEM_TRADE_CHILD) > ITEM_MASK_TRUTH ?
-            ITEM_MASK_TRUTH :
-            INV_CONTENT(ITEM_TRADE_CHILD) - 1,
-        INV_CONTENT(ITEM_TRADE_CHILD) >= ITEM_MASK_TRUTH || INV_CONTENT(ITEM_TRADE_CHILD) < ITEM_MASK_KEATON ?
-            ITEM_MASK_KEATON :
-            INV_CONTENT(ITEM_TRADE_CHILD) + 1,
```

Comment:
Same as before, not going to force you to change this now as it touches existing code, but it'd be really nice to get all the trade item code in z_kaleido_item.c moved over to hooks as well.